posts/2021-08-31-open-liberty-starter.adoc

```diff
@@ -0,0 +1,70 @@
+seo-description: Yet another way to get started with Open Liberty
+blog_description: Yet another way to get started with Open Liberty
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Open Liberty Starter
+Steven Zvonek <https://github.com/steven1046>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[#Intro]
+== Open Liberty Starter Application
+
+There is now another easy way to get started using Open Liberty. Visit the link:{url-prefix}/downloads/[Getting Started] page and select a few options to generate a starter application using Open Liberty. You can specify your application and project name, choose a build tool from either Maven or Gradle, and pick which version of Java SE, Jakarta EE, and MicroProfile your application will use. Then, just click `Generate Project` and you are ready for lift-off! 
+
+[link={url-prefix}/downloads]
+image::img/blog/starter_application.png[align="center"]
+
+[#Alternatives]
+== Other ways to start using Open Liberty
+
+There are a few other ways to get started with Open Liberty if you prefer instead:
+
+[#Docker]
+=== Docker image
+
+You can pull in the official Open Liberty docker image from link:https://hub.docker.com/_/open-liberty[Docker Hub] and build your application image on top of the Open Liberty image. There is a guide on using Docker with Open Liberty link:{url-prefix}/guides/containerize.html[here].
+
+=== Maven and Gradle
+You can also use Maven or Gradle to pull in Open Liberty as a dependency for your application. 
+
+==== Maven
+
+To get started using Open Liberty with Maven, add the following to your `pom.xml`, while specifying the version of Open Liberty that you want to use:
+
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>[21.0.0.8,)</version>
+    <type>zip</type>
+</dependency>
+----
+
+For more information on using Open Liberty and Maven, visit our link:{url-prefix}/guides/maven-intro.html[Maven Intro Guide].
+
+==== Gradle
+
+To get started using Open Liberty with Gradle, add the following to `build.gradle`, while specifying the version of Open Liberty that you want to use:
+
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[21.0.0.8,)'
+}
+----
+
+For more information on using Open Liberty and Gradle, visit our link:{url-prefix}/guides/gradle-intro.html[Gradle Intro Guide].
+
+=== Open Liberty Binary
+
+Download an Open Liberty release from our link:Getting Started[{url-prefix}/downloads/#runtime_releases_table_container] page. Then navigate to the `bin` directory and execute `./server create <server_name>` to create an Open Liberty server. Then, you can add in your application to the dropins directory, or add the path to your application in the newly created `<path_to_Open_Liberty_install>/servers/<server_name>/server.xml`.
```

Comment:
Add something regarding unzip/extract:
Download an Open Liberty release from our link:Getting Started[{url-prefix}/downloads/#runtime_releases_table_container] page **and extract the package.**

Reply:
Then navigate to the `bin` directory and execute `./server create <server_name>` to create an Open Liberty server, or use `./server start` to automatically create and start a `defaultServer`.

Reply:
Add links to docs, something like:
For more information regarding Liberty commands and configuration, visit the link:Command overview[{url-prefix}/docs/latest/reference/command/command-overview.html] and link:Server configuration overview[{url-prefix}/docs/latest/reference/config/server-configuration-overview.html] docs.